benchmarks/lambda_vs_nested_methods.rb

```diff
@@ -0,0 +1,23 @@
+require 'benchmark/ips'
+
+def nested_methods
+  def some_method
+  end
+end
+
+def lambda_method
+  some_lambda = lambda {|i|}
+end
+
+
+Benchmark.ips do |x|
+  x.report("nested methods") do
+    nested_methods
+  end
+
+  x.report("lambda") do
+    lambda_method
+  end
+
+  x.compare!
+end
```

Comment:
Can you add the benchmark results you get?  Ideally after a `__END__` line so they can just be pasted as comments at the end of the file.  That way we can see some results w/o having to run it ourselves :).